File: builds/android_ndk_jni_mk/java/com/github/sisong/ApkPatch.java
package com.github.sisong;

public class ApkPatch{
    // return TPatchResult, 0 is ok; patchFilePath file created by ZipDiff;
    public static native int patch(String oldApkPath,String patchFilePath,String outNewApkPath,
                                   long maxUncompressMemory,String tempUncompressFilePath,int threadNum);
}

File: builds/android_ndk_jni_mk/apk_patch.cpp
// apk_patch.cpp
// Created by sisong on 2019-08-22.
#include "apk_patch.h"

TPatchResult ApkPatch(const char *oldApkPath,const char *patchFilePath,const char *outNewApkPath,
                      size_t maxUncompressMemory,const char *tempUncompressFilePath,int threadNum){
    return ZipPatch(oldApkPath,patchFilePath,outNewApkPath,
                    maxUncompressMemory,tempUncompressFilePath,threadNum);
}
